diff --git a/src/optiondefs.h b/src/optiondefs.h
index f238209..5b55520 100644
--- a/src/optiondefs.h
+++ b/src/optiondefs.h
@@ -1534,6 +1534,11 @@ static struct vimoption options[] =
 				ISK_LATIN1
 #endif
 			    } SCTX_INIT},
+#ifdef __MVS__
+    {"ispflock",    "ifl",  P_BOOL|P_VI_DEF,
+			    (char_u *)&p_ifl, PV_NONE, NULL, NULL,
+			    {(char_u *)TRUE, (char_u *)0L} SCTX_INIT},
+#endif
     {"isprint",	    "isp",  P_STRING|P_VI_DEF|P_RALL|P_COMMA|P_NODUP,
 			    (char_u *)&p_isp, PV_NONE, did_set_isopt, NULL,
 			    {
